modules/selenium/src/test/java/org/testcontainers/junit/BrowserWebDriverContainerTest.java

```diff
@@ -51,34 +47,50 @@ public void provideDefaultNoProxyEnvironmentIfNotSet() {
     public void createContainerWithShmVolume() {
         try (
             BrowserWebDriverContainer webDriverContainer = new BrowserWebDriverContainer()
+                .withCapabilities(new FirefoxOptions())
         ) {
             webDriverContainer.start();
 
-            final List<InspectContainerResponse.Mount> mounts = webDriverContainer.getContainerInfo().getMounts();
-            assertEquals("Shm mounts present", mounts.size(), 1);
-
-            final InspectContainerResponse.Mount shmMount = mounts.get(0);
-            assertEquals("Shm mount source is correct", "/dev/shm", shmMount.getSource());
-            assertEquals("Shm mount destination is correct", "/dev/shm", shmMount.getDestination().getPath());
-            assertEquals("Shm mount mode is correct", shmMount.getMode(), "rw");
+            assertEquals("Only one shm mount present", countCorrectlyConfiguredShmVolumes(webDriverContainer), 1L);
+            assertEquals("No additional shm mounts present", countAnyShmVolumes(webDriverContainer), 1L);
         }
     }
 
     @Test
     public void createContainerWithoutShmVolume() {
         try (
-            BrowserWebDriverContainer webDriverContainer =  new BrowserWebDriverContainer<>()
-             .withSharedMemorySize(512 * FileUtils.ONE_MB)
+            BrowserWebDriverContainer webDriverContainer = new BrowserWebDriverContainer<>()
+                .withSharedMemorySize(512 * FileUtils.ONE_MB)
+                .withCapabilities(new FirefoxOptions())
         ) {
             webDriverContainer.start();
-            assertEquals("Shared memory size is configured", 512 * FileUtils.ONE_MB, webDriverContainer.getShmSize());
 
-            final long shmMountCount = webDriverContainer.getContainerInfo().getMounts()
-                .stream()
-                .filter(m -> "/dev/shm".equals(m.getSource()))
-                .count();
-            assertEquals("No shm mounts present", shmMountCount, 0L);
+            assertEquals("Shared memory size is configured",
+                512 * FileUtils.ONE_MB,
+                webDriverContainer.getShmSize());
+
+            assertEquals("No shm mounts present", countAnyShmVolumes(webDriverContainer), 0L);
         }
     }
 
+    private long countAnyShmVolumes(final BrowserWebDriverContainer container) {
```

Comment:
nit: if we change the result type to `Stream`, then we can reuse it in `streamCorrectlyConfiguredShmVolumes`, and the assertion remains readable:
```java
assertEquals("No shm mounts present", streamShmVolues(webDriverContainer).count(), 0L);
```